data/rinshin.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "Rinshin",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "**A human-level Programmer | Who wants to learn more!!",
+  "tags": ["Open Source", "Nextjs", "Linux", "Django", "Node.js"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/just_me_rinz" },
+    { "icon": "FaGithub", "url": "https://github.com/Rinshin-Jalal" },
+    { "icon": "FaLink", "url": "https://rinshin.me" }
+  ],
+  "avatar": "https://avatars.githubusercontent.com/u/88965873?v=4",
```

Use short image link.
```suggestion
  "avatar": "https://github.com/Rinshin-Jalal.png",
```